pkg/api/types.go

```diff
@@ -708,6 +712,14 @@ type ConfigMapVolumeSource struct {
 	Items []KeyToPath `json:"items,omitempty"`
 }
 
+// DockerVolume represents a volume that is backed by a Docker volume plugin. This depends on the plugin being already properly setup.
```

Comment:
I don't understand - the args only get used on `volume create` - using a volume just needs a name, doesn't it?

We need to decide what this driver will do.  Will it make a new volume (docker volume create) and then use it and then destroy it?  Or will it only use a volume that already exists?  Or both?